components/trakt/actions/remove-from-watchlist/remove-from-watchlist.mjs

```diff
@@ -0,0 +1,47 @@
+import trakt from "../../trakt.app.mjs";
+
+export default {
+  name: "Remove From Watchlist",
+  version: "0.0.1",
+  key: "trakt-remove-from-watchlist",
+  description: "Remove one of more items from watchlist. [See documentation here](https://trakt.docs.apiary.io/#reference/sync/remove-from-watchlist/remove-items-from-watchlist)",
+  type: "action",
+  props: {
+    trakt,
+    movies: {
```

Comment:
Looks like, `movies`and `tv shows` are being used in `add-to-watch` action. Could you please move it to `app`? Also, looks like the description Is wrong